src/xRetry.Reqnroll.v3/SkipException.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+
+// ReSharper disable once CheckNamespace
+// ReSharper disable once IdentifierTypo
+namespace Xunit
+{
+    /// <summary>
+    /// Do not use.
+    /// Exists purely as a marker to replicate the exception thrown by Xunit.SkippableFact that Reqnroll.xunit.v3
+    /// makes use of. That way we can intercept the exception that is throwing without also having our own runtime
+    /// plugin, or adding a direct dependency on either of these other libraries.
+    /// </summary>
+    internal class SkipException : Exception { }

```

Comment:
Since this is now built-in to xunit.v3, I don't think we need this. The code using it could reference the "real" class. For the old xunit v2 plugin, this was here as a marker class so that we didn't need to add an additional dependency.

Reply:
This might not even be used, I don't remember seeing it in the `TestGeneratorProvider`

Reply:
Removed